cirq/ops/parity_gates.py

```diff
@@ -30,20 +30,32 @@
 class XXPowGate(eigen_gate.EigenGate,
                 gate_features.TwoQubitGate,
                 gate_features.InterchangeableQubitsGate):
-    """The X-parity gate, possibly raised to a power.
+    r"""The X-parity gate, possibly raised to a power.
```

Comment:
Why the r? 
```suggestion
    """The X-parity gate, possibly raised to a power.
```

Reply:
Ignore me - this is fine. raw strings are best for backslash to be interpreted as a character.